### DISCUSSION TITLE
Stereo Pair

### DISCUSSION THREAD

**Discussion by Dunkelschunkel** [music-assistant/support#1337] [Q&A] [answered]
Good evening,

is it possible to set up my own stereo pair of symfonisk speakers via Universal Group Player like it is possible through the Sonos app?
What I have tried so far (and failed) is setting up the speakers in MA one with left channel the other with right channel audio only. After grouping them I do not have a dedicated left and right speaker. Both channels are either merged or just left or right but always played on all group players. I am running out of ideas.
Am I doing something wrong?

Cheers Dunkelschunkel

**Comment by marcelveldt** [comment]
Uhm why not create the stereo pair in sonos itself ?
Or you want to control them seperately ? Because that should work if you set the output channel to left and right respectively

**Reply by Dunkelschunkel** [reply]
@marcelveldt That is what I have been using previously. But on my journey of discovery I was wondering if the same can be achieved using Music Assistant. 
When skimming through the settings for each speaker it even mentions Stereo Pairs
![grafik](https://github.com/music-assistant/hass-music-assistant/assets/83595637/8f0135d1-b51b-4bf9-b918-473664e325f5)

But using 2 Sonos speakers, left and right, grouped together does not result in a true stereo pair

Dunkelschunkel

**Reply by Dunkelschunkel** [reply]
To confuse even further I tried the same setup with Airplay integration and not the Sonos one.
2 Sonos speakers via AirPlay, left  and right, grouped together **does** result in a good sounding proper stereo pair

![grafik](https://github.com/music-assistant/hass-music-assistant/assets/83595637/f3b00332-7a02-47e9-8e87-ec0765333ae2)

Dunkelschunkel

**Reply by marcelveldt** [reply, marked as answer]
yeah with airplay we have control over the exact audio stream that goes to the speaker. In case of grouped sonos devices, that ecosystem just sends the stereo stream to each speaker. So the only way to create a stereo pair is to either control those speakers from MA with airplay or create the stereo pair within the sonos ecosystem.

**Comment by Dunkelschunkel** [comment]
Thanks for shedding some light on this topic. I am probably going with airplay then.

Good night

**Reply by Dunkelschunkel** [reply]
just as a follow up for you.
I am now using a native stereo pair sonos setup. At least for this two speakers. Since they are fixed and are not moved between rooms that's the way to go. On other speakers it is easy to group them via Music Assistant, even the Stereo Pair can be part of Universal Player Groups for different scenarios.

Thanks for taking the time to engage in discussion. I do appreciate all your effort 

Dunkelschunkel
